kolibri/core/assets/src/views/CoreMenu/__tests__/CoreMenuOption.spec.js

```diff
@@ -0,0 +1,165 @@
+import { render, fireEvent, screen } from '@testing-library/vue';
+import VueRouter from 'vue-router';
+import CoreMenuOption from '../CoreMenuOption.vue';
+import '@testing-library/jest-dom';
+
+const sampleSubRoutes = [
+  { name: 'subRoute1', label: 'Sub Route 1' },
+  { name: 'subRoute2', label: 'Sub Route 2' },
+  { name: 'subRoute3', label: 'Sub Route 3' },
+];
+const sampleLink = 'https://mockurl.com/';
+const sampleIcon = 'add';
+
+const renderComponent = props => {
+  return render(CoreMenuOption, {
+    props: {
+      subRoutes: [],
+      ...props,
+    },
+    routes: new VueRouter(),
+  });
+};
+
+describe('CoreMenuOption', () => {
+  describe('subRoutes toggle', () => {
```

Comment:
Here we can have first a `smoke test` that just renders the component and check that it doesnt break [e.g.](https://github.com/EshaanAgg/kolibri/blob/4a3a4160420d8cb7b7fe39c96e455d122a5aa09f/kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus/__tests__/index.spec.js#L5)

Reply:
Will add the same!